.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+
+# See https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  Tests:
+    runs-on: ubuntu-latest
+    container: registry.opensuse.org/yast/head/containers/yast-ruby:latest
+
+    steps:
+
+    - name: Git Checkout
+      uses: actions/checkout@v2
+
+    # FIXME: optionally install the missing dependencies
```

Comment:
I'd remove those `FIXME`s, it's in the template to make it easier to adapt. If it is not needed (and won't be used in the future then it does not make much sense...)